src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -703,4 +733,105 @@ private void workAroundHttpClientCloneBug() {
             alignment.setCigar(new ArrayList<>(alignment.getCigar()));
         }
     }
+
+    /**
+     * Return a SAM string representation of this read.
+     */
+    private String createSAMString() {
+        final String SAM_FIELD_SEPARATOR = "\t";
+        final String SAM_ATTRIBUTE_SEPARATOR = ":";
+
+        final String contigName = getContig();
+        final StringBuilder sb = new StringBuilder();
+
+        sb.append(getName());
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(Integer.toString(ReadUtils.getSAMFlagsForRead(this)));
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(null == contigName ? SAMRecord.NO_ALIGNMENT_REFERENCE_NAME : contigName);
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(Integer.toString(getStart()));
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(Integer.toString(getMappingQuality()));
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(getCigar().toString());
+        sb.append(SAM_FIELD_SEPARATOR);
+
+        if (!isPaired()) {
+            sb.append("=");
```

Comment:
Good catch - none of the new tests caught this because of the quirky way SAMRecord does determines this value (it never consults the isPairedFlag,only the contig and mate contig names), and it was matching our output on all of the records since they all started out paired and were then transitioned to unpaired. I added a new test case specifically to trigger this clause..